### PR TITLE
Alex/mantine update & more!

### PR DESCRIPTION
**Pull request by alxsavard**
The new Mantine 7.X has a lot of breaking changes so there's quite a bit in here. 

- Updated to Mantine 7 (And Next 13) https://github.com/AlleyCorpNord/acn-radar/issues/88
- With Next 13, we have move to the new app router
- Fixed https://github.com/AlleyCorpNord/acn-radar/issues/90
- Partly fixed https://github.com/AlleyCorpNord/acn-radar/issues/89
- Took the liberty of installing prettier

### Mantine migration
Matine migrated to css modules with 7.x and the styles are almost exactly the same but not quite (if you pixel-peep). Since we were using a lot of the default styling I didn't put too much effort into making it exactly the same as it was before. It's close enough that I think it'll be fine!

### Migration to the app router
Since mantine only supports client side components, I had to add some extra layers in some places just to split mantine stuff from what could/had to run on the server. The obvious example is the blips page now being separated from the blips components. 

Next also removed the shallow routing ability (https://github.com/vercel/next.js/discussions/48110) from the new app router so I had to use the native history api which works the same way, but it just looks weird haha. 

#### Notes
I was not sure what the /templates/blips.tsx page was for so I didn't touch it. 

image

### CODE REVIEW

**Review by ysael — APPROVED** [review]
Nice! 🔥

**Comment by ysael** [comment]
image
but the deploy preview seems broken 🤔

**Comment by alxsavard** [comment]
Seems like it's just the base path. If you navigate to https://deploy-preview-92--acn-radar.netlify.app/blips/expo it works 🤔. 

I'll look into it, thanks!

**Comment by alxsavard** [comment]
Fixed 👌

**Review by julienblin — APPROVED** [review]
Did not review in details but this looks overall LGTM.
Awesome @alxsavard - thanks a lot!

File: src/helpers/Parsers.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/**
+ * Returns the text from the first paragraph of an HTML string
+ */
+export function getFirstParagraphText(htmlString: string) {
+  const regex = /<p>(.*?)<\/p>/;
+  const match = regex.exec(htmlString);
+  if (match) return match[1];
+  return "";
+}
```
Comment (roblinton) [inline src/helpers/Parsers.ts:9]: DOM parsing with regexs takes me waaay back to the days where I didn't know about xpath.  😆 

Ah... the memories...

Given this is a "friendly" internal app there's probably nothing egregiously wrong here security-wise... but in a more strict environment I don't think we'd get away with this.

**Review by roblinton — APPROVED** [review]

**Pull request merged**